zone/lua_stat_bonuses.cpp

```diff
@@ -1097,7 +1097,7 @@ int32 Lua_StatBonuses::GetSkillReuseTime(int idx) const {
 
 int32 Lua_StatBonuses::GetSkillDamageAmount(int idx) const {
 	Lua_Safe_Call_Int();
-	return self->SkillDamageAmount[idx];
```

Comment:
Probably should make sure we don't access out of bounds here as well, this would just prevent script writers from foot gunning (crash)